Nodejs/Product/TestAdapterImpl/TestFrameworks/Jasmine/jasmine.js

```diff
@@ -80,43 +81,59 @@ function initializeJasmine(Jasmine, projectFolder) {
     return instance;
 }
 
-function getSuiteName(spec) {
-    var suiteName = spec.getSpecName(spec);
-    if (suiteName && suiteName.length > 0) {
-        return suiteName.substr(0, suiteName.length - spec.description.length - 1);
-    }
-    return suiteName;
+/**
+ * @param {jasmine.Suite} suite
+ * @param {object[]} testList
+ * @param {string} testFile
+ */
```

Comment:
Would it be useful to doc the return type?

Reply:
The compiler was smart enough to figure out that was `void`, so it didn't need help.